sdr/Make.defs

```diff
@@ -0,0 +1,21 @@
+############################################################################
+# apps/sdr/Make.defs
```

Comment:
should we move to math?

Reply:
> should we move to math?

At first I was thinking the same, but while liquid-dsp has a lot of math functions, it has more functions related to SDR and also sdr could house some other libraries and programs in the future. My suggestion is to stick with it.

Reply:
Ok.